src/features/commentsSlice/thunks.ts

```diff
@@ -0,0 +1,36 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import {
+  createComment,
+  deleteComment,
+  getPostComments,
+} from '../../api/comments';
+import { Comment } from '../../types/Comment';
+import { commentsActions } from './commentsSlice';
+
+const loadComments = createAsyncThunk('comments/fetch', (postId: number) => {
+  return getPostComments(postId);
+});
+
+const addComment = createAsyncThunk('comments/add',
+  async (comment: Omit<Comment, 'id'>) => {
+    try {
+      const newComment = await createComment(comment);
+
+      return newComment;
+    } catch {
+      throw new Error('Failed to add comment');
+    }
+  });
+
+const removeComment = createAsyncThunk('comments/remove',
+  async (commentId: number, { dispatch }) => {
+    try {
+      await dispatch(commentsActions.localeRemoveComment(commentId));
+
+      return await deleteComment(commentId);
```

Comment:
please check the difference between `return Promise.resolve()` `Promise.resolve()` and just `return`
```suggestion
      return deleteComment(commentId);
```